contracts/modules/RiskManager.sol

```diff
@@ -62,21 +63,38 @@ contract RiskManager is IRiskManager, BaseLogic {
         } else {
             // Uniswap3 TWAP
 
-            // The selected fee level is the first existing pool found in the following order.
-            // This may need to be corrected by a governance action if there is insufficient liquidity in the pool.
+            // The uniswap pool (fee-level) with the highest in-range liquidity is used by default.
+            // This is a heuristic and can easily be manipulated by the activator, so users should
+            // verify the selection is suitable before using the pool. Otherwise, governance will
+            // need to change the pricing config for the market.
 
-            uint24 fee;
-            if (IUniswapV3Factory(uniswapFactory).getPool(underlying, referenceAsset, 3000) != address(0)) fee = 3000;
-            else if (IUniswapV3Factory(uniswapFactory).getPool(underlying, referenceAsset, 500) != address(0)) fee = 500;
-            else if (IUniswapV3Factory(uniswapFactory).getPool(underlying, referenceAsset, 10000) != address(0)) fee = 10000;
-            else revert("e/no-uniswap-pool-avail");
```

Comment:
There is now also a 0.01% fee option on Uniswap V3 on eth mainnet

Reply:
Will add, thx!